packages/api/test/codegen/languages/typescript/index.test.ts

```diff
@@ -123,6 +125,11 @@ describe('typescript', () => {
 
     it('should work against the petstore', assertSDKFixture('@readme/oas-examples/3.0/json/petstore.json', 'petstore'));
 
+    it(
+      'should work against metrotransit',
+      assertSDKFixture('@api/test-utils/definitions/metrotransit.json', 'metrotransit'),
+    );
```

Comment:
Also added a new SDK fixture for @kanadgupta's [metrotransit project](https://metrotransit.readme.io/reference/get_nextrip-agencies) test project.